Spatial.tscn

```diff
@@ -217,51 +246,72 @@ __meta__ = {
 
 [node name="VBoxContainer" type="VBoxContainer" parent="Control/Dialogs/MainMenu/ScrollContainer"]
 margin_right = 220.0
-margin_bottom = 284.0
+margin_bottom = 368.0
 size_flags_horizontal = 3
 
 [node name="LoadPath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
 margin_right = 220.0
 margin_bottom = 40.0
 rect_min_size = Vector2( 0, 40 )
-text = "Open Markers File"
+text = "Toggle Marker Visibility"
 
-[node name="SavePath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+[node name="HSeparator5" type="HSeparator" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
 margin_top = 44.0
 margin_right = 220.0
-margin_bottom = 84.0
+margin_bottom = 48.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="ImportPath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+margin_top = 52.0
+margin_right = 220.0
+margin_bottom = 72.0
+text = "Import Marker Pack"
+
+[node name="HSeparator6" type="HSeparator" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+margin_top = 76.0
+margin_right = 220.0
+margin_bottom = 80.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="SavePath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+margin_top = 84.0
+margin_right = 220.0
+margin_bottom = 124.0
 rect_min_size = Vector2( 0, 40 )
 text = "Save Markers File"
 
 [node name="HSeparator" type="HSeparator" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
-margin_top = 88.0
+margin_top = 128.0
 margin_right = 220.0
-margin_bottom = 92.0
+margin_bottom = 132.0
 
 [node name="PointEditor" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
-visible = false
-margin_top = 96.0
-margin_right = 213.0
-margin_bottom = 136.0
+margin_top = 136.0
+margin_right = 220.0
+margin_bottom = 176.0
 rect_min_size = Vector2( 0, 40 )
 text = "Editor Panel"
```

Comment:
Is this duplicate "Editor Panel" button used anymore? It does not seem to have any signals attached to it.

Reply:
It is not. Removing.